### ISSUE TITLE
Problem in the video output format

### ISSUE THREAD

**Issue by alexdiehl** [puemos/hls-downloader-web-extension#85]
**Describe the bug**
When HLS downloader finish the download, export as .ts to save

**Expected behavior**
Save file as .mp4.
Note: Before the April 24 update, I was able to download videos from this same site
Video link:
https://svbpt-sambavideos.akamaized.net/voda/_definst_/amlst%3Astg%3B3436%2C8982%2Ca210c617a181fbab2a02decaccd565af%3Bhidden32np%3B3WUIQJBEWIDX2AUBHLTAK2UIANTDMKNBSLYS4OCIZJMWFPFGB77RRIBMP4APXL3CSMP5GZJQJ2NREOA24NFDNKDIW453ORQILZEX7CCBDMGG2RLHM54T343RKAEUGFNTLAZ4TAH6BMZJZ35LIRH76DXZGJJJQFFL2UWUYDJYRZRWAT4T3YMIFKO5GQJBLBTBTDEF4GKTJPC4JFIKQFB6DVWF67RDSKHDGLVMCHPIFHFAA7PX6EWGWMYSUU7OSBJPH72X7IELATKKSFRURX2AQT5SY3OK3FRPNJWXVBYOVO53VHXLNRNCP5IFYUZ4OI7KMJRH3FPK2JIVWRRG2BTIM6ZZXTAA2T43DPR27MTCECJDOQWF/playlist.m3u8?sts=st=1588018991~exp=1588041191~acl=/*~hmac=9198f17f8c133b99a2bc35f1994141800231259a1aaf804ae075b52283331bad

**Screenshots**
![image](https://user-images.githubusercontent.com/25112515/80418080-3483e400-88ad-11ea-8e77-d59835b462c6.png)
![image](https://user-images.githubusercontent.com/25112515/80418251-7ca30680-88ad-11ea-8426-98af4ad7728b.png)

**Desktop
 - OS: Windows 10
 - Browser Chrome
 - Version 81.0.4044.122

**Comment by puemos** [comment]
Hey! quick question, are you able to watch the files?
Technically these are not mp4 files, they are ts files (https://en.wikipedia.org/wiki/MPEG_transport_stream).
You can just change the file extension to `.mp4`

**Comment by alexdiehl** [comment]
First of all, thanks for the quick return

If I change the file extension to .mp4 I can't play the video, just like when it has a .ts extension.
Before the update to 2.0 when I was going to save the video the .mp4 extension already came and I was able to play the video normally.

**Comment by puemos** [comment]
it could be related to another issue that was solved in v2.1.1 (AES-128 encryption support) that is still pending on the chrome store, so you have 3 options:

1. Wait until Google will approve the new version (1-2 days)
2. Use the Firefox version
3. Use the unpacked version and install it by yourself (not recommended)

**Comment by alexdiehl** [comment]
Thanks! I will use Firefox version until the new version of chrome.

**Comment by alexdiehl** [comment]
Just for your knowledge. I downloaded the extension for firefox. The final file extension remains .ts, however, now I can play the video normally. Thanks

**Comment by puemos** [comment]
FYI: @alexdiehl chrome just release v2.1.1

**Comment by shape55** [comment]
Here is the enhancement request for the mp4 conversion feature: https://github.com/puemos/hls-downloader/issues/213.
